syft/frameworks/torch/tensors/interpreters/native.py

```diff
@@ -157,9 +199,6 @@ def _is_parameter(self):
         """
         return isinstance(self, syft.hook.torch.nn.Parameter)
 
-    def copy(self):
```

Comment:
will we still have a copy method for native tensor?

Reply:
Yes in abstract.py